llvm_ir_dataset_utils/tools/RemoveFunctionBodyPass/install

```diff
@@ -0,0 +1,3 @@
+mkdir build && cd build
+cmake -GNinja ..
```

Comment:
This script doesn't seem to be particularly useful and I don't think it needs to be in-tree.